pkgs/development/python-modules/dataclass-wizard/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pythonOlder
+, pythonAtLeast
+, pytimeparse
+, pyyaml
+, pytestCheckHook
+, pytest-mock
+, typing-extensions
+, withOptionalDependencies ? false
+}:
+
+buildPythonPackage rec {
+  pname = "dataclass-wizard";
+  version = "0.22.2";
+
+  src = fetchFromGitHub {
+    owner = "rnag";
+    repo = "dataclass-wizard";
+    rev = "v${version}";
+    hash = "sha256-Ufi4lZc+UkM6NZr4bS2OibpOmMjyiBEoVKxmrqauW50=";
+  };
+
+  propagatedBuildInputs = [
+  ] ++ lib.optionals (pythonOlder "3.9") [
+    typing-extensions
+  ] ++ lib.optionals withOptionalDependencies (builtins.attrValues {
+    inherit (passthru.optional-dependencies) all;
+  });
+
+  passthru.optional-dependencies = {
+    all = [
+      pytimeparse
+      pyyaml
+    ];
```

```suggestion
    timedelta = [
      pytimeparse
    ];
    yaml = [
      pyyaml
    ];
```
https://github.com/rnag/dataclass-wizard/blob/e3fe0760422613eaf79e0236840a50f27caec064/setup.py#L75-L78